src/tools/interop/idt/README.md

```diff
@@ -299,14 +265,58 @@ $ idt capture -h
 usage: idt capture [-h] [--platform {Android}] [--ecosystem {DemoExtEcosystem...
 ```
 
+> **IMPORTANT:** Note the following runtime expectations of ecosystems:  
```

Comment:
I might have just missed this, but what is analyze_capture? And what does async aware mean in this context?

Reply:
Analyze capture is the function of ecosystems that is expected to run continuously and provide helpful ~"real time" output to the user. Async aware in this context would mean that the function should not block the event loop for excessive periods of time, to allow execution of other platform and ecosystem coroutines while the capture executes.

Reply:
Clarified!